tar/tarsnap.1-man.in

```diff
@@ -889,6 +889,9 @@ In create and extract modes,
 \fB\%tarsnap\fP
 will list each file name as it is read from or written to
 the archive.
+In delete mode,
```

Comment:
Thanks, looks good!  Could you please rebase this to combine the two commits into one?